include/gba/defines.h

```diff
@@ -92,7 +94,15 @@
 #define RGB_CYAN RGB(0, 31, 31)
 #define RGB_WHITEALPHA (RGB_WHITE | 0x8000)
 
+// Some functions are strictly inline asm
 #define NAKED __attribute__((naked))
+
+// Silence IDE warnings
+#if __GNUC__ >= 4
+#define USED __attribute__((used))
+#else
+#define USED
+#endif
```

Comment:
I'm not sure I like this. Surely the linker is properly sorting these out, and I don't think it's worth littering the repo with `USED` prefixes just to silence warnings in some IDEs